docs/user-guide/host-settings.md

```diff
@@ -0,0 +1,193 @@
+# Host Settings
+This page contains settings for Linux servers, it agnostic of the Linux
+distribution. The commands use the `ip` utility from the `iproute2` package.
+These settings do not persist across reboots. Consult the documentation from
+the Linux distribution for guides on how to persist the settings in for example
+[Netplan](https://documentation.ubuntu.com/server/explanation/networking/configuring-networks/index.html) or 
+[Network Manager](https://networkmanager.dev/docs/admins/). For additional
+details consult the [kernel bonding
+driver](https://www.kernel.org/doc/html/latest/networking/bonding.html)
+documentation.
+
+## MCLAG / ESLAG
+TODO - does this active/active?
+
+The multi-chassis LAG architecture is a way to reduce the number of failure points
+in a network architecture. At the physical layer an MCLAG topology is a single
+server connected to two different switches and those switches are connected.
+For a total of 3 physical connections.
```

Comment:
I think mentioning physical conns that includes conn between switches is very misleading.

Reply:
TODO: remove the summation